source/includes/_chat_summary.md

```diff
@@ -0,0 +1,174 @@
+## Chat Summary
+
+### Resource Overview
+
+
+| Method | URI Format |
+|---|---|
+| GET | /client_reports/chat_summary/[gmaid]?[query_params]|
+
+### API Name: Chat Summary
+
+### Summary
+
+This API can be used by customers who have our chat widget on their site. The API has metrics on the chat along with traffic information (Paid Campaign/Organic).
+
+### Usage
+Use GET to retrieve information from the Chat Summary API. Data can be returned for a GMAID by a specific date range determined by start date and end date.
+
+Subtotals will be returned for each Master Campaign ID with chats in that date range.
```

Comment:
Perhaps "chat activity" instead of "chats".

Reply:
This has been updated to "chat activity".